darwin/torch/dataset.py

```diff
@@ -96,23 +95,35 @@ def __getitem__(self, index: int):
             category_id : int
                 The single label of the image selected
         """
-        img = load_pil_image(self.images_path[index])
+        img = self.get_image(index)
         if self.transform is not None:
             img = self.transform(img)
 
+        target = self.get_target(index)
+
+        return img, target
+
+    def get_target(self, index: int):
+        """Returns the classification target
+        """
```

Comment:
Honestly not sure if a formatter would make this change. In any case, let's make sure we run the formatter!

```suggestion
        """
        Returns the classification target
        """
```